openapi.yml

```diff
@@ -743,6 +769,106 @@ components:
           type: string
           example: 2008/0545166T
           description: An identifier from the Police National Computer (PNC)
+    Sentence:
+      type: object
+      properties:
+        dataSource:
+          type: string
+          enum: [NOMIS, NDELIUS]
+          example: NOMIS
+          description: Which upstream API the sentence originates from
+        dateOfSentencing:
+          tags: nomis, ndelius
+          type: string
+          format: date
+          example: 2009-09-09
+          description: Date of sentencing
+        description:
+          tags: nomis, ndelius
+          type: string
+          example: Young Offender Inst - >=12 mths
+          description: Description of the sentence
+        isActive:
+          tags: nomis, ndelius
+          type: boolean
+          example: true
+          description: Whether the sentence is active
+        isCustodial:
```

Comment:
Is this missing tags?

Reply:
Intentionally left this one out. 
The flag exists in nDelius only, we always set this to true for Nomis within our own API.

Feels like tagging both, or only one API in this case would be deceiving.
Thoughts?

Reply:
Ahhhhh okay, I think let's leave it out!